ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -512,6 +518,25 @@ tests:
         TEST_FUNC=test_networking
       PACKET_PLAN: m3.large.x86
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-kube-api-net-suite-periodic
```

Comment:
move it right after the presubmit job, to have it better organized

Reply:
Done